### PR TITLE
Fix strings to translate

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: public/views/create.html [hunk 1]
```diff
@@ -132,9 +132,8 @@
             </label>
 
             <label  for="bws" class="oh">
-              <span translate>Specify Bitcore Wallet Service URL
-                <input type="text" id="bwsurl" name="bwsurl" ng-model="bwsurl">
-              </span>
+              <span translate>Specify Bitcore Wallet Service URL</span>
```
Comment (matiu) [inline public/views/create.html]: It should be "Wallet Service URL" only

File: public/views/import.html [hunk 2]
```diff
@@ -85,9 +85,8 @@
               </label>
 
               <label  for="bws" class="oh">
-                <span translate>Specify Bitcore Wallet Service URL
-                  <input type="text" id="bwsurl" name="bwsurl" ng-model="bwsurl">
-                </span>
+                <span translate>Specify Bitcore Wallet Service URL</span>
```
Comment (matiu) [inline public/views/import.html]: see above

File: public/views/import.html [hunk 3]
```diff
@@ -213,9 +211,8 @@
               <div class="large-12 columns">
 
                 <label  for="bws" class="oh">
-                  <span translate>Specify Bitcore Wallet Service URL
-                    <input type="text" id="bwsurl" name="bwsurl" ng-model="bwsurl">
-                  </span>
+                  <span translate>Specify Bitcore Wallet Service URL</span>
```
Comment (matiu) [inline public/views/import.html]: see above

File: public/views/preferencesAdvanced.html [hunk 4]
```diff
@@ -27,7 +27,7 @@
  
     <li  ng-click="$root.go('preferencesBwsUrl')">
       <i class="icon-arrow-right3 size-24 right text-gray"></i>  
-      <span translate>Bitcore Wallet Service</span>
+      <span>Bitcore Wallet Service</span>
```
Comment (matiu) [inline public/views/preferencesAdvanced.html]: Wallet Service URL

**Comment by cmgustavo** [comment]
Updated

**Pull request merged**